packages/cluster-ui/src/statementsTable/statementsTable.module.scss

```diff
@@ -24,7 +24,7 @@
   }
```

Comment:
Do I need to change anything here?

Reply:
Nope, these are overrides for specific column widths in case you need them.